src/lib/get-marked-with-highlighter.ts

```diff
@@ -1,20 +1,18 @@
 import { getLanguage, highlight } from 'highlight.js';
 import marked, { MarkedOptions } from 'marked';
 
-export const getMarked = (options: MarkedOptions) => {
+export const getHighlightRenderer = (options: MarkedOptions) => {
 	const renderer = options.renderer ?? new marked.Renderer();
```

Comment:
It doesn't make sense to have this in combination with `marked.use()` anymore. If there's `options.renderer`, then we need to call `marked.use({ renderer: options.renderer })` instead.

Reply:
uhm yeah. because marked.use will merge renderer automatically ?